### ISSUE TITLE
[BUG] - BetterPlaylist with short videos and nextVideoDelay 0 value - hangs on.

### ISSUE THREAD

**Issue by thedoubler** [jhomlala/betterplayer#417]
**Describe the bug**
Having short videos, when setting nextVideoDelay to 0 player stops playing after a second video.

**To Reproduce**

1) Download sample app.
2) Change the **nextVideoDelay** property to 0 seconds in playlist_page.dart
3) Replace setupData() with : 

```  
Future<List<BetterPlayerDataSource>> setupData() async {
    var list = [
      "https://media2.giphy.com/media/CweWL0UDpTjo4QLo42/giphy.mp4",
      "https://media2.giphy.com/media/V7u69ZL5IWsZqSUVcy/giphy.mp4",
      "https://media.giphy.com/media/xTiTnJeYiGPnwDt5pm/giphy.mp4",
    ];

    list.forEach((element) {
      _dataSourceList.add(
        BetterPlayerDataSource(BetterPlayerDataSourceType.network, element),
      );
    });

    return _dataSourceList;
  }
```
4) Run and go to Playlist section and hit play button.

**Actual behavior**
Playing videos will stop after second video.

**Expected behavior**
Playing of the videos should play without interruption if nextVideoDelay = 0

**Better Player version**
- Version: ^0.0.63

**Smartphone:**
 - Device: Galaxy J6
 - OS: Android 10

**Comment by jhomlala** [comment]
@thedoubler This will be fixed with next release.

**Comment by thedoubler** [comment]
That's great. Do you know about when the next release will be?

**Comment by jhomlala** [comment]
Fixed in 0.0.64.
